bundle/manifests/instaslice-operator.clusterserviceversion.yaml

```diff
@@ -214,13 +242,44 @@ spec:
                     drop:
                     - ALL
               securityContext:
-                runAsNonRoot: true
+                runAsNonRoot: false
```

Comment:
Just to make sure everything works, has this been tested?

Reply:
Yes let's leave this as it requires root privileges now.